pyproject.toml

```diff
@@ -78,6 +79,11 @@ test-command = "pytest -v --no-cov {project}/tests"
 # don't build PyPy wheels, install from source instead
 skip = "pp*"
 
+[[tool.cibuildwheel.overrides]]
+select = "cp313t-*"
+build-frontend = "build; args: --no-isolation"
+before-build = "bash {package}/scripts/cibw_before_build.sh"
+
```

Comment:
Also another comment here, I'm unclear if only the before-build line or other lines should be removed when Cython 3.1 is out.

Reply:
Isn't this a list? Perhaps, the script could be stuffed right inside this config?